plugins/processors/enum/enum_test.go

```diff
@@ -91,6 +93,22 @@ func TestMapSingleBoolValue(t *testing.T) {
 	assertFieldValue(t, 1, "true_value", fields)
 }
 
+func TestMapSingleIntValue(t *testing.T) {
+	mapper := EnumMapper{Mappings: []Mapping{{Field: "int_value", ValueMappings: map[string]interface{}{"13": int64(14)}}}}
+
+	fields := calculateProcessedValues(mapper, createTestMetric())
+
+	assertFieldValue(t, 14, "int_value", fields)
+}
+
+func TestMapSingleUintValue(t *testing.T) {
+	mapper := EnumMapper{Mappings: []Mapping{{Field: "uint_value", ValueMappings: map[string]interface{}{"50": int64(5)}}}}
+
+	fields := calculateProcessedValues(mapper, createTestMetric())
+
+	assertFieldValue(t, int64(5), "uint_value", fields)
+}
+
```

Comment:
can we test mapping this to a string? seems like a common use case.

Reply:
Makes sense, I removed the existing tests and added table tests to cover it more exhaustively. :)